test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
You're right.  This is a good point.  (Of course, I've had my qualms about the derivable stuff in FL too, wondering if their API is simply too granular, but that's besides the point at the moment.)

> We could *also* dispatch to `contains`

I think I would prefer that.  If a type provide both `contains` and `indexOf`, we should definitely prefer to dispatch to the `contains` version.